contrib/mandoc/st.c

```diff
@@ -56,7 +56,7 @@ LINE("-isoC-tcor1",	"ISO/IEC 9899/TCOR1:1994 (\\(lqISO\\~C90, Technical Corrigen
 LINE("-isoC-tcor2",	"ISO/IEC 9899/TCOR2:1995 (\\(lqISO\\~C90, Technical Corrigendum 2\\(rq)")
 LINE("-isoC-99",	"ISO/IEC 9899:1999 (\\(lqISO\\~C99\\(rq)")
 LINE("-isoC-2011",	"ISO/IEC 9899:2011 (\\(lqISO\\~C11\\(rq)")
-LINE("-isoC-2024",	"ISO/IEC 9899:2024 (\\(lqISO\\~C23\\(rq)")
+LINE("-isoC-2023",	"ISO/IEC 9899:2024 (\\(lqISO\\~C23\\(rq)")
```

Comment:
I'd accept both to avoid chicken and egg problems that may otherwise arise.

Reply:
Deprecated comment for the old one + notice in src/UPDATING? Accept both until 16.0 release?

Reply:
Alternatively, what if I MFC changing all the C24s that can MFC so we don't have this problem?

Reply:
I'd add isoC2023 like you have and add a deprecated comment. I'd commit those changes here and the rest of the tree. then start on doing the vendor import to get the rest. I'd merge it to 14 after 14.3 is released (since it's a minor thing that doesn't change the rendered man pages and re@ dislikes churn that doesn't make things better). Catch me on irc/discord/etc and we can work out the logistics since there's a lot of details that matter and it's hard to catch them all on a github pull request. They are documented in the handbook, but it can be a little confusing in places (which I'd love to see improved, but I've read it so many times I'm blind to it, though I also know it's confusing).